src/music/models.py

```diff
@@ -1,3 +1,35 @@
+from django.contrib.auth import get_user_model
 from django.db import models
 
-# Create your models here.
+from core.models import BaseModel
+
+
+class Track(BaseModel):
+    title = models.CharField(max_length=200)
+    length = models.IntegerField(null=True, blank=True)
+    artist = models.CharField(max_length=200, blank=True)
```

Comment:
In the Track model, consider making artist a foreign key to an Artist model if you have many tracks from the same artist. This will reduce data redundancy and simplify queries.